ubuntusetup/setup.sh

```diff
@@ -10,11 +10,11 @@ ubuntusetup() {
     if [ -e .paperbenni/setup/programs.txt ]; then
         echo "program list found"
     else
-        echo "no list found"
+        echo "no list found" >&2
```

Comment:
Ensuring output redirects accordingly can help yourself or other users who might want to log or otherwise process the output (like logging only the errors) or do some other ghastly thing.